cmd/rook/ceph/osd.go

```diff
@@ -326,8 +326,11 @@ func removeOSDs(cmd *cobra.Command, args []string) error {
 		return errors.Wrapf(err, "failed to parse --preserve-pvc flag")
 	}
 
+	forceRemovalCallback := func(x int) (bool, bool) {
+		return forceOSDRemovalBool, false
```

Comment:
nit: For readability it helps to assign variable names when passing the parameters.
```suggestion
		exitIfNotSafe := false
		return forceOSDRemovalBool, exitIfNotSafe
```

Reply:
done